website/docs/docs/deploy/dbt-cloud-job.md

```diff
@@ -5,126 +5,68 @@ description: "You can enable continuous integration (CI) to test every single ch
-- You must set up a [deployment environment](/docs/collaborate/environments/dbt-cloud-environments) 
-- Your deployment environment must be on dbt version 1.0 or higher
-
-## Features
-
-dbt Cloud simplifies job scheduling, documentation, logging, automation, and alerting. These features make it easy to deploy jobs confidently and build observability into your processes.
-
-### Scheduler 
-
-The dbt Cloud's Scheduler is a powerful tool that helps data teams optimize their data transformation workflows, makes it easier to run dbt in production environments, governs data, automates your dbt jobs, and empowers teams to get results faster with less effort. 
-
-Use the scheduler to improve the efficiency of your data transformation process, while also gaining greater visibility into its progress. You can use custom scheduling for your jobs, which allows you to flexibly run jobs to meet your company's data freshness needs. The dbt Cloud Scheduler gives you power, while abstracting away complexity.
-
-Refer to [Scheduler](/docs/deploy/job-scheduler) for more info on how the scheduler works. 
-
-<Lightbox src="/img/docs/dbt-cloud/deployment/deploy-scheduler.jpg" width="100%" title="An overview of a dbt Cloud job run"/>
-
-### Notifications
-
-Set up job notifications to inform you via email or a selected Slack channel when a job run succeeds, fails, or is canceled, enabling you to respond quickly and proactively.  
-
-Refer to [job notifications](/docs/deploy/job-notifications) for more info. 
-
-### Host and share docs
-
-You can set up, view, and share your project documentation in order to streamline your team's collaboration and productivity. Invite teammates to dbt Cloud to collaborate and share documentation. 
-
-Refer to [build and view your docs](/docs/collaborate/build-and-view-your-docs) for more info.
-
-### Access logs
-
-You can view or download in-progress and historical logs for your dbt runs, making it easy for anyone on the team to debug errors more efficiently.
-
-<Lightbox src="/img/docs/dbt-cloud/deployment/access-logs.jpg" width="85%" title="Access logs for run steps" />
-
-### Model timing 
-> Available on [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) dbt Cloud accounts on the [Team or Enterprise plans](https://www.getdbt.com/pricing/).
-
-The model timing dashboard on dbt Cloud displays the composition, order, and time taken by each model in a job run. The visualization appears for successful jobs and highlights the top 1% model durations. This helps you identify bottlenecks in your runs, so you can investigate them and potentially make changes to improve their performance. 
-
-You can find the dashboard on the **Run Overview** page. 
-
-<Lightbox src="/img/docs/dbt-cloud/model-timing.jpg" width="85%" title="The model timing tab displays the top 1% of model durations and visualizes model bottlenecks" />
-
```

Comment:
we may need to create a new page for this "Model timing" section

Reply:
Yes! let's create a new section for "Run Visibility". Can we put the model timing and [run history](https://docs.getdbt.com/docs/deploy/dbt-cloud-job#run-history) sections in there. Run history is new since you opened this PR.